formatTest/unit_tests/expected_output/lineComments.re

```diff
@@ -150,7 +150,8 @@ type t3 = (int, int); // End of line on (int, int)
 
 type variant =
   | X(int, int) // End of line on X
-  | Y(int, int); // End of line on Y // Comment on entire type def for variant
+  | Y(int, int); // End of line on Y
```

Comment:
I'd say this one was an improvement.

Reply:
I also believe that this improvement lends itself better to idempotency. (No proof of that but just combining multiple things into one thing  as it did before seems to pose challenges).